src/main/java/cart/repository/ShoppingCartRepository.java

```diff
@@ -0,0 +1,13 @@
+package cart.repository;
+
+import cart.service.dto.CartResponse;
+import java.util.List;
+
+public interface ShoppingCartRepository {
+
+    void addProduct(long memberId, long productId);
+
+    void removeProduct(long cartId);
+
+    List<CartResponse> findAllProduct(long memberId);
```

Comment:
```suggestion
    List<CartResponse> findAllProductByMemberId(long memberId);
```

위에도 유사한 리뷰를 남겼는데 조회는 조건에 따라 결과값이 다른 경우가 많아서, 조건을 `by{}` 로 명시해줄 때 읽기가 더 편리하더라구요. 어떻게 생각하시나용?~

Reply:
by{}로 표현해주는 것은 확실히 메서드를 호출하는 곳에서 더 잘 읽힐 수 있을 것 같아요. 

저는 `by{}`로 조건을 명시해주는 것도 좋긴한데, 이미 파라미터 네이밍으로 충분히 표현이 된다고 생각해요.
조건이 늘어나면 늘어날수록 메서드 이름이 늘어나면 이상할 것 같기도 하고요.

그래서 저는 메서드 네이밍에 어떠한 동작을 할 것인지만 표시해요. 조건은 파라미터에 있기만 해도 잘 읽히는 편이어서요.